include/uv.h

```diff
@@ -1554,11 +1554,13 @@ union uv_any_req {
 };
 #undef XX
 
+typedef void(*uv_loop_cb)(uv_loop_t*);
 
 struct uv_loop_s {
   /* User data - use this for whatever. */
   void* data;
   /* Loop reference counting. */
+  uv_loop_cb on_watcher_queue_updated;
```

Comment:
I think this PR will need to target `master` instead of `v1.x`.

Reply:
@cjihrig fixed!

Reply:
Should this field be initialized in `uv_loop_init()`?